action/protocol/staking/candidate_center_extra.go

```diff
@@ -21,6 +21,9 @@ func (cb *candBase) clone() *candBase {
 	for bucket, cand := range cb.selfStkBucketMap {
 		clone.selfStkBucketMap[bucket] = cand.Clone()
 	}
+	for identity, cand := range cb.identifierMap {
```

Comment:
Is it a bug? we need to take a closer look at whether it's a hard fork.

Reply:
this is used by the 1.8.4 bug fix
we can maybe define a new `clone2`, or run a fullsync test